packages/botonic-core/src/handoff.js

```diff
@@ -126,15 +126,28 @@ export async function storeCaseRating(session, rating) {
   return resp.data
 }
 
-export async function getAvailableAgents(session, queueId) {
+export async function getAvailableAgentsByQueue(session, queueId) {
```

Comment:
please add this to index.d.ts
Be aware that I removed queueId's question mark because it makes no sense to omit this argument
```
export declare function getAvailableAgents(
  session: Session
): Promise<{ agents: {email: string, attending_count: number}[] }>

export declare function getAvailableAgentsByQueue(
  session: Session,
  queueId: string
): Promise<{ agents: string[] }>

```

Reply:
please update documentation in docs top folder